x-pack/plugins/aiops/server/routes/queries/get_simple_hierarchical_tree.ts

```diff
@@ -0,0 +1,267 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { max } from 'd3-array';
+// import { omit, uniq } from 'lodash';
```

Comment:
Can remove this commented line

Reply:
I'll leave it in because it's part of another commented part around `categories` which we might need later.